scipy/signal/tests/test_signaltools.py

```diff
@@ -224,16 +223,16 @@ def test_convolve_method_large_input(self):
         # This is really a test that convolving two large integers goes to the
         # direct method even if they're in the fft method.
         for n in [10, 20, 50, 51, 52, 53, 54, 60, 62]:
-            z = np.array([2**n], dtype=np.int64)
+            z = np.array([2 ** n], dtype=np.int64)
```

Comment:
all changes to `**` are wrong, we don't want spaces around `**`, please undo everywhere

Reply:
Apologies for this. I committed straight from pycharm with optimise code on.